### PR TITLE
Add option to output postgres log, and clean docker image when done

### PR DESCRIPTION
**Pull request by JLockerman**
Just like it says in the title. Add
```
-print-logs true
```
to print the postgres logs in the `end_to_end` tests

### CODE REVIEW

File: pkg/pgmodel/end_to_end_tests/main_test.go [hunk 1]
```diff
@@ -91,6 +100,12 @@ func withDB(t testing.TB, DBName string, f func(db *pgxpool.Pool, t testing.TB))
 	})
 }
 
+func withSuperuser(t testing.TB, DBName string, f func(db *pgxpool.Pool, t testing.TB)) {
```
Comment (cevian) [inline pkg/pgmodel/end_to_end_tests/main_test.go:103]: doesn't seem to be used?

File: pkg/internal/testhelpers/containers.go [hunk 2]
```diff
@@ -106,8 +106,20 @@ func dbSetup(DBName string, superuser SuperuserStatus) (*pgxpool.Pool, error) {
 	return dbPool, nil
 }
 
+type stdoutLogConsumer struct {
+}
+
+func (s stdoutLogConsumer) Accept(l testcontainers.Log) {
+	if l.LogType == testcontainers.StderrLog {
+		fmt.Print(l.LogType, " ", string(l.Content))
```
Comment (cevian) [inline pkg/internal/testhelpers/containers.go:114]: shouldn't this be Println (i.e. don't you need to add line breaks)

Comment (JLockerman) [reply pkg/internal/testhelpers/containers.go:114]: @cevian `l.Content` always has a trailing `\n`, if we use `Println` we get double-whitespace between the lines

File: pkg/pgmodel/end_to_end_tests/main_test.go (hunk 1 above)
Comment (JLockerman) [reply pkg/pgmodel/end_to_end_tests/main_test.go]: must've removed that test. Ah well, it was too brittle anyway

**Pull request merged**